maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -153,7 +160,7 @@ public void execute() throws MojoExecutionException {
 			}
 
 			// Expand Jar into target/classes
-			expandJar(bndJar, classesDir);
+			expandJar(bndJar, classesDir, buildContext);
```

Comment:
I am wondering if we can just change expandJar to be non-static, then we don't have to pollute the method signature with buildContext.